src/main/java/io/vertx/ext/mongo/MongoClient.java

```diff
@@ -172,6 +187,21 @@ MongoClient updateCollectionWithOptions(String collection, JsonObject query, Jso
 
   Future<@Nullable MongoClientUpdateResult> updateCollectionWithOptions(String collection, JsonObject query, JsonObject update, UpdateOptions options);
 
+  /**
+   * Use an aggregation pipeline to update documents in the specified collection, specifying options and return the handler with MongoClientUpdateResult result
+   *
+   * @param collection  the collection
+   * @param query  query used to match the documents
+   * @param update aggregation pipeline used to describe how documents will be updated
+   * @param options options to configure the update
+   * @param resultHandler will be called when complete
+   */
+  @Fluent
+  MongoClient updateCollectionWithOptions(String collection, JsonObject query, JsonArray update, UpdateOptions options,
+                                          Handler<AsyncResult<@Nullable MongoClientUpdateResult>> resultHandler);
+
+  Future<@Nullable MongoClientUpdateResult> updateCollectionWithOptions(String collection, JsonObject query, JsonArray update, UpdateOptions options);
```

Comment:
missing doc

Reply:
ping

Reply:
You'll have to be more specific.
I've already added the following javadoc (commit https://github.com/vert-x3/vertx-mongo-client/pull/254/commits/c7abe3ae5781f4107aa239c057c94ac1459293c2):
```java
  /**
   * Like {@link #updateCollectionWithOptions(String, JsonObject, JsonArray, UpdateOptions, Handler)} but returns a {@code Future} of the asynchronous result
   */
```
Is there something else?

Reply:
my bad, I did not spot it